thread/source/lib.c

```diff
@@ -10,7 +10,7 @@ void *memset(void * ptr, int value, unsigned int num){
 }
 
 int strcomp(char* s1, char* s2, unsigned int size){
-	for(int i = 0; i < size; i++){
+	for(int i = 0; i < size*2; i++){	// doubling size brings this in line with strcopy() and fixes the "Rei" string randomly replacing "Ver" in other places.
```

Comment:
Yeah I haven't seen any problems yet and I have had the updated arm9.bin this generates on my system since about 5am or so. The few people I have asked to test it in the thread haven't reported any adverse affects either.